examples/logger.py

```diff
@@ -186,10 +189,10 @@ def receivedSlot(self, channelName: str, content: Any):
             if isinstance(content, str):
                 self.addLog(content)
             else:
-                print("The message for the channel log should be a string.")
+                logger.error("The message for the channel log should be a string.")
         else:
-            print(f"The message was ignored because "
-                  f"the treatment for the channel {channelName} is not implemented.")
+            logger.error("The message was ignored because "
```

Comment:
I just realized it now, and I think you can remove `receivedSlot()` becuase we don't use the `log` channel more.

Reply:
I knew that but just leave it. I will delete receivedSlot.

Reply:
I'm sorry I should have told you earlier😅

Reply:
Anytime.